src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -255,26 +256,42 @@ protected override bool ProcessRequest(IRequestInfo info)
 				return ProcessLevedRTInfo(info, localPath);
 			else if (localPath.StartsWith("localhost/", StringComparison.InvariantCulture))
 			{
-#if __MonoCS__
```

Comment:
The essence of what I did here was to extract the platform-dependent path modification into one method, and the process of keeping the result if the resulting file exists into another.